corehq/pillows/synclog.py

```diff
@@ -116,7 +116,8 @@ def process_change(self, change):
 
 
 def mark_last_synclog(domain, user, app_id, build_id, sync_date, latest_build_date, device_id,
-        device_app_meta, commcare_version=None, build_profile_id=None, save_user=True):
+                      device_app_meta, commcare_version=None, build_profile_id=None, fcm_token=None,
+                      fcm_token_timestamp=None, save_user=True):
```

Comment:
Looks like we missed updating a use of mark_last_synclog, its just above this.

Reply:
looks like that gets called only for batch processing. Please let QA know so that they can test it and they should also keep a note so there are aware of this in future.

Reply:
> Looks like we missed updating a use of mark_last_synclog, its just above this.

Correct me If I am wrong, this pillow `UserSyncHistoryProcessor` is used for sync operations. This does not contain FCM details and hence I believe no change needed here.

Reply:
Offlined that the other call didn't need an update since the missing arguments were optional keyword arguments.